src/metabase/pulse/render/png.clj

```diff
@@ -73,16 +74,66 @@
           (.getSubimage image 0 0 content-width (.getHeight image))
           image)))))
 
+(defn- get-supported-glyphs
+  "Get a list of the unicode chars supported by the font `font-name`, if it can be found."
+  [font-name]
+  (style/register-fonts-if-needed!)
+  (let [env (GraphicsEnvironment/getLocalGraphicsEnvironment)
+        fonts (.getAllFonts env)
+        font ^Font (some #(when (= font-name (.getName ^Font %)) %) fonts)]
+    (when font
+      (let [char-range (range 0x0000 0xFFFF)]
+        (filter #(let [code (int %)]
+                   (.canDisplay font code)) char-range)))))
+
+(def ^:private lato-supported-chars
+  (into #{} (map char) (get-supported-glyphs "Lato Regular")))
+
+#_
+(def ^:private non-latin-regex
+  (re-pattern "[^A-Za-z0-9!@#$%^&*()_+\\-=\\[\\]{}|;:'\",.<>?/`~\\s\\t\\n\\r]"))
```

Comment:
I still think this is too narrow; Lato supports cyrillic and greek chars.

Reply:
disregard that comment, I did not notice `#_` :)

Reply:
It's left there as a commented out form. Perhaps I should just remove it.